readme.md

```diff
@@ -3,7 +3,10 @@
 <h1 align="center">logdown.js</h1>
 
 <div align="center">
-<img src="http://travis-ci.org/caiogondim/logdown.js.svg?branch=master" alt="Travis CI"> <img src="http://img.badgesize.io/caiogondim/logdown.js/master/dist/logdown.min.js?compression=gzip" alt="lib size"> <img src="https://codecov.io/gh/caiogondim/logdown.js/branch/master/graph/badge.svg" alt="codecov"> <a href="https://www.npmjs.com/package/logdown"><img src="https://img.shields.io/npm/v/logdown.svg" /></a>
+[![Travis CI](http://travis-ci.org/caiogondim/logdown.js.svg?branch=master)](https://travis-ci.org/caiogondim/logdown.js?branch=master)
+![Lib Size](http://img.badgesize.io/caiogondim/logdown.js/master/dist/logdown.min.js?compression=gzip)
+[![Codecov](https://codecov.io/gh/caiogondim/logdown.js/branch/master/graph/badge.svg)](https://codecov.io/gh/caiogondim/logdown.js)
+[![npm](https://img.shields.io/npm/v/logdown.svg)](https://www.npmjs.com/package/logdown)
```

Comment:
Is that working?
Can't see a proper preview [here](https://github.com/caiogondim/logdown.js/blob/01568bce7fd30d604f37521b2448a5c5a4cf68f5/readme.md)

Reply:
Hm. Looks like github does not want to render markdown inside html tags